spec/spec_helper.rb

```diff
@@ -8,15 +8,16 @@
 require 'webmock/rspec'
 require 'rspec/its'
 
+require 'pry'
+
 Dir[File.join(File.dirname(__FILE__), 'support', '**', '*.rb')].sort.each do |file|
   require file
 end
 
 RSpec.configure do |config|
   config.mock_with :rspec
-end
+  config.filter_run_when_matching :focus
```

Comment:
Just so I know, is this a better of focusing tests compared to the deletion that was made in `Rakefile`?

Reply:
I don't know, that's not my change. @gerhard ?

Reply:
@pivotal-pmital feel free to revert this or do something different if the change is inappropriate.

Reply:
@pivotal-pmital yes, this is the recommended way of focusing on specs in rspec.

This config was already in place, I have just removed the `end` and the `focus` sub-task (it was duplicating the config). Everything is captured in this commit: https://github.com/pivotal/LicenseFinder/pull/671/commits/47880a54f098ed78a4f97c99ca7e05664107093c